docs/README.md

```diff
@@ -1,64 +1,70 @@
+When the Broadcaster app sends an event, it contains a payload with the following fields.
 
-| Field Name                                | Description                                                                                                                                                  | Type    |
-| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- |
-| `IdSku`                                   | SKU ID in VTEX                                                                                                                                               | string  |
-| `ProductId`                               | Product ID in VTEX                                                                                                                                           | long    |
-| `An`                                      | Account Name in VTEX, shown in the store’s VTEX Admin url.                                                                                                   | string  |
-| `IdAffiliate`                             | Affiliate ID generated automatically in the configuration.                                                                                                   | string  |
-| `DateModified`                            | Date when the item was updated.                                                                                                                              | string  |
-| `IsActive`                                | Identifies whether the product is active or not. If `true` the product/SKU is active                                                                         | boolean |
-| `StockModified`                           | Identifies that the inventory level has been altered. If `false`, the inventory level has not been changed.                                                  | boolean |
-| `PriceModified`                           | Identifies that the price has been altered. If `false`, the product/SKU price has not been changed.                                                          | boolean |
-| `HasStockKeepingUnitModified`             | Identifies that the product/SKU registration data has changed, such as name, description, weight, etc. If `true`, the product/SKU registration data changed. | boolean |
-| `HasStockKeepingUnitRemovedFromAffiliate` | Identifies that the product is no longer associated with the trade policy. If `true`, the trade policy has changed.                                          | boolean |
+| Field Name | Description | Type |
+| - | - | - |
+| `IdSku` | SKU ID on VTEX. | string |
+| `ProductId`| Product ID on VTEX. | long |
+| `An`| Account Name on VTEX, shown in the store’s VTEX Admin URL. | string |
+| `IdAffiliate` | Affiliate ID generated automatically in the configuration. | string |
```

Comment:
As discussed on Slack, `ProductId` and `IdAffiliate` only seem to be part of the event payload if the notification comes from an affiliate/seller. There's also a `SellerChain` string field sent in this scenario.